lib/vtls/mbedtls.c

```diff
@@ -697,7 +680,7 @@ static CURLcode mbed_connect_step1(struct Curl_cfilter *cf,
       ret = mbedtls_pk_parse_keyfile(&backend->pk, ssl_config->key,
                                      ssl_config->key_passwd,
                                      mbedtls_ctr_drbg_random,
-                                     &backend->ctr_drbg);
+                                     &rng.drbg);
```

Comment:
This line references `rng` outside of the same set of `#ifdef` used for declaring the variable. Looks fragile/wrong?

Reply:
The `#if MBEDTLS_VERSION_NUMBER >= 0x04000000` is the same behavior. But the defines *are* different than up above (so can be confusing for future developers). I'll change this define so it's clearer. I'll make the changes tonight.